app/views/users/show.html.erb

```diff
@@ -1 +1,39 @@
-A user's details with a list of posts
\ No newline at end of file
+<%= javascript_include_tag "show_more" %>
+<div class="user-wrapper">
+  <div class="user-container">
+    <div class="user-list">
+      <span class="photo">
+        <%= image_tag(@user.photo, size: "100x100" ) %>
+      </span>
+      <span class="user-info">
+        <span>
+          <%= link_to @user.name, user_path(user_id: @user.id, format: :html) %>
+        </span>
+        <span>
+          number of posts: <%= @user.posts_counter ? @user.posts_counter : 0 %>
+        </span>
+      </span>
+    </div>
+    <div class="bio">
+      <span> Bio </span>
+      <span> <%=@user.bio%> </span>
+    </div>
+    <ul class="posts">
+      <% @user.posts.each do |post| %>
+      <li>
+        <span>
+          <%= link_to "Post #: #{post.id}" , user_posts_path(user_id: @user.id)
+          %>
+        </span>
+        <span> <%= post.text %> </span>
+        <span
+          >comments: <%= post.comments.count %>&nbsp; likes: <%=
+          post.likes.count %></span
+        >
+      </li>
+      <% end %>
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you use the `#recent_posts` method to render the user's 3 most recent post for the `app/views/users/show.html.erb` page. This was supposed to be mentioned previous reviewers, I suggest you implement it before merging your PR as it is one of requirements. i.e.:
```ruby
  ...
    <ul class="posts">
      <% @user.recent_posts.each do |post| %>
      <li>
        <span>
          <%= link_to "Post #: #{post.id}" , user_posts_path(user_id: @user.id)
          %>
        </span>
        <span> <%= post.text %> </span>
        <span
          >comments: <%= post.comments.count %>&nbsp; likes: <%=
          post.likes.count %></span
        >
      </li>
      <% end %>
    </ul>
  ...
```